datastore.go

```diff
@@ -52,11 +53,13 @@ func NewDatastore(path string, opts *Options) (*Datastore, error) {
 		return nil, err
 	}
 
-	return &Datastore{
-		accessor: &accessor{ldb: db, syncWrites: true},
+	closeLk := sync.RWMutex{}
```

Comment:
nit: you can do `accessor: &accessor{ldb: db, syncWrites: true, closeLk: &sync.RWMutex{}}` if you want, but nbd at all.

Reply:
was trying to be clear that we're ending up with the single canonical memory location for the mutex defined here as a local variable. I also have no strong opinion for what's preferable here.